rewriter/ClassNew.cc

```diff
@@ -10,9 +10,39 @@ using namespace std;
 
 namespace sorbet::rewriter {
 
-vector<ast::ExpressionPtr> rewriteAsClassDef(core::MutableContext ctx, ast::Assign *asgn) {
+// Is this expression a `Magic.bind` call we added from the `ClassNew` rewriter on sends?
```

Comment:
Instead of using `<Magic>` bind, can we leave it as just `T.bind` but set the `isRewriterSynthesized` bit on the `Send`?

I'd love to have this change not involve any diff to `GlobalState.cc` nor `resolver.cc`